### ISSUE TITLE
feat(web-worker): Scroll event should send more information

### ISSUE THREAD

**Issue by Serginho** [angular/angular#28964]
# 🚀 feature request

### Relevant Package
@angular/platform-webworker

### Description
It's very common to use `scrollTop` and `scrollHeight` properties when a scroll event is set. As it is in the Keyboard event to use the key. For example: To paint a custom scroll rail or to paginate a list.

It would be nice to have those properties in the scroll event when we are using platform-webworker.

### Describe the solution you'd like
It's very easy, I can upload a PR if you agree with this.

Basically, we only have to add the same behavior like `serializeMouseEvent`, but with the scroll event.

https://github.com/angular/angular/blob/9f7a9c607ef4c14c280cc84fad79fe5db8ad6fc2/packages/platform-webworker/src/web_workers/ui/event_dispatcher.ts#L38-L89

### Describe alternatives you've considered
Extending WorkerRenderer and creating a new EventDispatcher

Tell me if you like or dislike this.

**Comment by robwormald** [comment]
The issue with this approach (and this is one of those reasons that workers are still experimental, because we don't really have a great solution yet!) is that accessing properties like `scrollTop` and `scrollHeight` [(and about a hundred others...)](https://gist.github.com/paulirish/5d52fb081b3570c81e3a) may trigger a reflow, so eagerly reading and serializing those properties is likely to be a bit of a performance nightmare.

**Comment by Serginho** [comment]
I agree with you, but for me. But, for this aproach those properties are important like keyboardEvent and `keyCode`. I know it's only a hotfix but better than do thing in the UI thread. When do you use scroll event without `scrollTop`? Hardly ever.

**Comment by jelbourn** [comment]
Closing all issues related to web-workers since Angular's experimental web-worker renderer has been deprecated and it now being removed completely in Angular v11.
